node/Dockerfile.amd64

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015-2021 Tigera, Inc. All rights reserved.
```

Comment:
I think you need this change in ARM64 too; that has BPF support now.

For PPC64/ARM32 etc, we don't support BPF yet but please try to make sure the change doesn't break those.  You could add nsenter/mount to those too then we're ready if we add BPF later